examples/features/csm_observability/README.md

```diff
@@ -0,0 +1,30 @@
+# CSM Observability
+
+This examples shows how to configure CSM Observability for a binary, and shows
```

Comment:
A hyperlink to CSM somewhere in this page?

Reply:
I agree, but the documentation is currently internal/confidential and there's no blog post about it yet or any public facing docs. Perhaps I can mention to refer to the CSM user guide?

Reply:
Yes, please.

Reply:
Added comment at the end of this paragraph. The UG is not published yet, will add a hyperlink once the public preview UG goes out.